docs/src/main/asciidoc/rest-client.adoc

```diff
@@ -155,6 +155,34 @@ If you don't rely on the JSON default, it is heavily recommended to annotate you
 It will allow to narrow down the number of JAX-RS providers (which can be seen as converters) included in the native executable.
 ====
 
+=== Query Parameters
+
+If the GET request requires query parameters you can leverage the `@QueryParam('query-name')` annotation instead of (or in addition to) the `@PathParam`. Path and query parameters can be combined, as required, as illustrated in a mock example below.
```

Comment:
Minor gripe but I would make it:
```suggestion
If the GET request requires query parameters you can leverage the `@QueryParam("parameter-name")` annotation instead of (or in addition to) the `@PathParam`. Path and query parameters can be combined, as required, as illustrated in a mock example below.
```

Reply:
Ok, that's done, and squashed